itou/templates/admin/job_applications/jobapplication_change_form.html

```diff
@@ -21,7 +21,7 @@
                        class="danger js-with-confirm"
                        name="transition_cancel"
                        value="Annuler"
-                       {% if not original.can_be_cancelled %} disabled data-bs-toggle="tooltip" data-bs-placement="top" title=" Une fichie salarié doit probablement être supprimée avant de pouvoir annuler cette candidature " {% endif %}>
+                       {% if not original.can_be_cancelled %} disabled data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Une fichie salarié doit probablement être supprimée avant de pouvoir annuler cette candidature" {% endif %}>
```

Comment:
```suggestion
                       {% if not original.can_be_cancelled %} disabled data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Une fiche salarié doit probablement être supprimée avant de pouvoir annuler cette candidature" {% endif %}>
```
au passage (idéalement dans un autre commit, avec la suppression des espaces :angel: )